package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "hawkular-opentracing",
+  "version": "0.0.1",
+  "description": "Hawkular OpenTracing implementation",
+  "keywords": [
+    "opentracing",
+    "distributed tracing",
+    "tracing",
+    "hawkular",
+    "application performance"
+  ],
+  "author": "Pavol Loffay",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/hawkular/hawkular-apm#readme",
```

Comment:
I took it from here https://github.com/bucharest-gold/automatic-pancake/blob/master/package.json#L22. Readme part was removed :+1: